go/signedexchange/signedexchange_test.go

```diff
@@ -229,7 +229,7 @@ func TestSignedExchangeBannedCertUrlScheme(t *testing.T) {
 	}
 }
 
-func TestVerify(t *testing.T) {
+func createTestExchange(t *testing.T) (e *Exchange, s *Signer, certBytes []byte) {
```

Comment:
@hajimehoshi Any thoughts on using named return values here?
https://tour.golang.org/basics/7 discourages using it on long func, but I'm open here.